packages/next/server/config.ts

```diff
@@ -812,17 +809,9 @@ export default async function loadConfig(
       )
     }
 
-    if (userConfig.target && !targets.includes(userConfig.target)) {
+    if (userConfig.target) {
```

Comment:
```suggestion
    if (userConfig.target && userConfig.target !== 'server') {
```
We don't want to error if the default value is passed as this could break in various places.

Reply:
Aren't we deprecating the whole option, not just `target: "serverless"`?: https://nextjs.org/blog/next-12#:~:text=making%20target%20deprecated%20in%20Next.js

Reply:
We don't want to hard error for a valid value since plugins and such may be providing this automatically